lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -141,6 +146,10 @@ def command=(value)
     task.application_name = value
   end
 
+  def comment=(value)
+    task.comment = value
```

Comment:
Comment was removed from the updated PR, so when the other comes up, the validation can come with it.